Automation.sc

```diff
@@ -18,23 +18,24 @@
 
 Automation {
     var <>length,
-        <>presetDir,
-        <>onPlay, <>onStop, <>onSeek,
-        <>playLatency = 0.1, <>seekLatency = 0.1,
-        <>doStopOnSeek = false, 
-        <>onEnd,
-        <>verbose = true,
-        <>server,
-        <>doDefer = true,
-        <doRecord = false,
-        <clients,
-        <>gui,
-        startTime = -1, startOffset = 0,
-        semaphore = nil,
-        playRoutine,
-        playStatus = false,
-        playDoReschedule = true,
-        playLastStopped = 0;
+	<>presetDir,
+	<>onPlay, <>onStop, <>onSeek,
+	<>playLatency = 0.1, <>seekLatency = 0.1,
+	<>doStopOnSeek = false, 
+	<>onEnd,
+	<>verbose = true,
+	<>server,
+	<>doDefer = true,
+	<doRecord = false,
+	<clients,
+	<>gui,
+	<>hideLoadSave = false,
```

Comment:
Please swap the logic to 'showLoadSave = true' instead. The double negation "hide is false, hence they are shown" isn't as straightforward as a simple 'showLoadSave = true'.